docs/WooCommerceAdditionalTokens.md

```diff
@@ -0,0 +1,143 @@
+:::
+
+#### Example token config
+```
+"BTC_OFFCHAIN";"Lightning BTC";"payment";""
+"USDt";"USDt (Liquid Theter)";"payment";"https://example.com/wp-content/uploads/2021/01/usdt.png"
+"eKr";"eKrona (Liquid Asset)";"promotion";""
+```
+
+After saving you will see each asset to be available as payment method. You can enable/disable them like any other payment method. They won’t have any settings itself for now though (everything is configured by the CSV data). But you can use them together with e.g. woocommerce payment plugins to allow discounts for certain payment methods etc.
+
+// todo: add screenshot
+
+
+## Common WooCommerce use-cases with other plugins
+
+### Use-case 1: limit product to a region/shipping zone
+Free plugin used: [Country Based Restrictions for WooCommerce](https://wordpress.org/plugins/woo-product-country-base-restrictions/)
+After installing and activating the plugin go to a product in “Product data” block there is a new tab “Country restrictions”. You can configure the restrictions you want there.
+
+Example configuration:   
+// todo: add screenshot
+
+### Use-case 2: (Promotion) products should have free shipping
+This is doable with woocommerce out of the box (no plugins needed):
+
+1. In shipping settings add a new shipping class e.g. “free-shipping”
+2. On your shipping zones / shipping method config you need to make sure you set the rate to be 0 for that shipping class but also that “cost” is empty or 0. And your “no shipping class cost” is set to the normal rate (using flat-rate as example):   
+   // todo: add screenshot
+3. In product settings "Product data" block you have a tab "Shipping", there you set the above created "Free-shipping" class and it will taken care during checkout.   
+   // todo: add screenshot
+
+### Use-case 3: limit product payment methods
+E.g. allow only a special token payment for promotional product(s)
+
+Free plugin used: [Conditional Payments for WooCommerce](https://wordpress.org/plugins/conditional-payments-for-woocommerce/)
+
+This plugin provides a condition rules builder where you can enable/disable available payment methods for products. See example config in the screenshot:   
+// todo: add screenshot
+
+### Use-case 4: discount per payment method
+
+Free plugin used: [Discounts Per Payment Method for WooCommerce](https://wordpress.org/plugins/woo-payment-discounts/)
+
+In the now available “Discount per Payment” setting in your WooCommerce settings you have a list of all payment methods and can give percentage or fixed discounts.
+
+// todo: add screenshot
+
+### Use-case 5: make sure promotional products can only be purchased exclusively
+
+This is needed because promotion token payment methods need to override the product price with 1 (per quantity). Otherwise a user could mix normal products with promo products during checkout, which you want to avoid.
```

Comment:
```suggestion
This is needed because payment methods based on a selected Currency, Asset, Altcoin or Token need to override the product price with 1 (per quantity). Otherwise a user could mix normal products with promo products during checkout, which you want to avoid.
```

Reply:
I will copy this but enrich it a bit with more information to avoid confusion.